components/compliance-service/api/tests/03_reports_spec.rb

```diff
@@ -712,6 +712,19 @@ def reporting
     assert_equal(Reporting::Reports, resp.class)
     assert_equal(9, resp['reports'].length)
 
+    # Get a specific report; ensure only the control with the specified filter tag is returned
+    res = GRPC reporting, :read_report, Reporting::Query.new(id: '44024b50-2e0d-42fa-a57c-dddddddddddd',       
+        filters: [
+            Reporting::ListFilter.new(type: 'control_tag:tag1', values: ['value-1']),
+        ]
+    )
+    assert_equal(Reporting::Report, res.class)
+    assert_equal(Google::Protobuf::RepeatedField, res['profiles'].class)
+    if res['profiles'].is_a?(Google::Protobuf::RepeatedField)
```

Comment:
my docker es (non-studio) was being real punky so not sure if this test will pass or not/didn't run it locally

Reply:
this test is failing, so i'll get my local makefile dev figured out and see what's going on

Reply:
Same comment here, we don't need this if statement: https://github.com/chef/automate/pull/1672#discussion_r330807475